src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts

```diff
@@ -118,20 +117,22 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
         this.rgwMultisiteService.getSyncStatus().pipe(
           tap((data: any) => {
             this.loading = false;
-            this.replicaZonesInfo = data['dataSyncInfo'];
-            this.metadataSyncInfo = data['metadataSyncInfo'];
-            [this.realm, this.zonegroup, this.zone] = data['primaryZoneData'];
+            if (data === 'Unable to get sync status') {
+              this.showMultisiteCard = false;
+              this.syncCardLoading = false;
+              return;
+            } else {
+              this.replicaZonesInfo = data['dataSyncInfo'];
+              this.metadataSyncInfo = data['metadataSyncInfo'];
+              if (this.replicaZonesInfo.length === 0) {
+                this.showMultisiteCard = false;
+                this.syncCardLoading = false;
+              }
+              [this.realm, this.zonegroup, this.zone] = data['primaryZoneData'];
+            }
           })
         )
```

Comment:
```suggestion
        ),
        catchError((error) => doThisWhenErrorHappens 
```

Reply:
for eg: https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.ts#L153